libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -30,6 +29,20 @@ import {
 } from 'botbuilder-core';
 import { TeamsInfo } from './teamsInfo';
 
+/**
+ * Adds support for Microsoft Teams specific events and interactions.
+ * @remarks
+ * Developers may handle activities sent from Microsoft Teams via two methods:
+ *  1. Overriding methods starting with `handle...()` (e.g. `handleTeamsTaskModuleFetch()`) .
+ *      * The only way to handle Invoke activities is to override these methods.
+ *  2. Overriding methods starting with `on..` and *not* ending in `..Event()` (e.g. `onTeamsMembersAdded()`)
+ *  3. Passing callbacks to methods starting with `on..` *and* ending in `...Event()` (e.g. `onTeamsMembersAddedEvent()`),
+ *      to stay in line with older {@link ActivityHandler} implementation, or instead
```

Comment:
`, or instead` seems like it should be removed.

Reply:
Whoops! I switched around #2 and #3 and forgot to fix that part. Re-pushed with fix/update.